lib/routes/account.js

```diff
@@ -411,8 +411,9 @@ module.exports = function (
 
         checkIsBlockForced()
           .then(() => customs.check(request, email, 'accountLogin'))
-          .catch(checkUnblockCode)
+          .catch(extractUnblockCode)
```

Comment:
Despite its name, this function does not actually check the unblock code, it just extracts the code for future checking by a separate function.